docs/src/fs.rst

```diff
@@ -175,6 +175,7 @@ API
         On Windows libuv uses `CreateFileW` and thus the file is always opened
         in binary mode. Because of this the O_BINARY and O_TEXT flags are not
         supported.
+        On Windows `uv_fs_XXX` functions use utf-8 encoding.
```

Comment:
Right now you have this in the documentation for `uv_fs_open()`. I don't think that's the best place for it. The first paragraph of the file might be a better place.

Reply:
Ok @cjihrig.I will do the changes accordingly and make a pull request.

Reply:
Hi @cjihrig,
Added the Note to the first paragraph of the file.

Reply:
Thanks @bzoz , @cjihrig for suggestions. I've made the required changes and pushed the commit.